dlt/destinations/dataset/dataset.py

```diff
@@ -171,6 +175,29 @@ def row_counts(
         # Execute query and build result dict
         return self(query)
 
+    def list_load_ids(self, status: Union[int, list[int]] = 0, limit: int = 10) -> list[str]:
```

Comment:
This should return an SupportsReadableRelation, the user can convert this into a query string or run the .df() or arrow() functions on it. Check how it is done in the row_counts() method.